{{cookiecutter.repo_name}}/{{cookiecutter.app_name}}/models.py

```diff
@@ -14,6 +14,10 @@
 class {{ model.strip() }}(TimeStampedModel):
     """
     TODO: replace with a brief description of the model.
+
+    TODO: Add either a negative or a positive PII annotation to the end of this docstring.  For more
+    information, see OEP-30:
+    https://open-edx-proposals.readthedocs.io/en/latest/oep-0030-arch-pii-markup-and-auditing.html
```

Comment:
note for reviewers: I want to keep this un-annotated to force unit tests to fail on the new repo.